eng/scripts/job-matrix/README.md

```diff
@@ -165,11 +165,124 @@ once for each instance of a language version.
 
 The processing order is as follows:
 
-1. The base matrix is generated
-1. The imported base matrix is generated
+Given a matrix and import matrix like below:
+```
+{
+    "matrix": {
+        "$IMPORT": "example-matrix.json",
+        "JavaVersion": [ "1.8", "1.11" ],
+        "TestFromSource": [ false, true ]
+    }
+}
+
+### example-matrix.json to import
+{
+    "matrix": {
+      "operatingSystem": [ "windows", "linux" ],
+      "client": [ "netty", "okhttp" ]
+    },
+    "include": [
+        {
+          "operatingSystem": "mac",
+          "client": "netty"
+        }
+    ]
+}
+```
+
+1. The base matrix is generated (sparse in this example):
+    ```
+    {
+      "storage_18": {
```

Comment:
Where does storage come from in your example?

Reply:
fixed